release.sh

```diff
@@ -109,6 +109,7 @@ rm -f trunk/CHANGELOG.txt
 rm -f trunk/CONTRIBUTING.md
 rm -f trunk/CODE_OF_CONDUCT.md
 rm -f trunk/release.sh
+rm- f trunk/.vscode
```

Comment:
Command is invalid. It should be `rm -f trunk/.vscode` (space should be before `-`).

Reply:
Fixed my typo https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/36/commits/a42c09537c67fa7194f6c059a64c0da77e56d485